client/state/selectors/get-billing-transaction-date-filter-values.js

```diff
@@ -20,7 +20,7 @@ export default createSelector(
 			return [];
 		}
 
-		const result = times( 6, ( n ) => {
+		const result = [ 0, 1, 2, 3, 4, 5 ].map( ( n ) => {
```

Comment:
Alternatively something like 

```
[...new Array(6).keys()].map( /* ... */ )
```

but using literals seemed so much simpler and easier to grasp 😄

Reply:
`[ ...Array( 6 ) ].map( ( v, k ) => k )` is straightforward enough IMHO, but I'm happy to leave this as you suggested. 

In tests, simple and more readable is always better 👍